bcolz/tests/test_carray.py

```diff
@@ -1603,6 +1603,11 @@ def test12(self):
         self.assertTrue(type(cr) == np.ndarray)
         assert_array_equal(cr, nr, "eval does not work correctly")
 
+    def test13(self):
+        """Testing eval() with columnar [shape = (n, 1)] arrays"""
```

Comment:
Also note that this test won't trigger the bug unless `self.N` is large enough that chunking is required.